supplementary_style_guide/style_guidelines/formatting.adoc

```diff
@@ -104,6 +104,42 @@ Write all titles and headings, including the titles of product documentation gui
 * _Configuring the node port service range_
 * _How to perform an unsupported conversion from a RHEL-derived Linux distribution to RHEL_
 
+[[commands-in-code-blocks]]
+== Commands in code blocks
```

Comment:
+1 For Technical examples

Reply:
Hm, I'd say it fits rather well under the "Formatting" section, since "Technical examples" primarily seems to deal with guidelines for the content itself rather than its markup. 

Also, the "User-replaced values" parts, which deal with a very much related topic, including code block content  (which is why I initially bunched up the new section with them), are under Formatting as well.